src/video/wayland/SDL_waylandwindow.c

```diff
@@ -622,6 +704,11 @@ Wayland_SetWindowBordered(_THIS, SDL_Window * window, SDL_bool bordered)
 {
     SDL_WindowData *wind = window->driverdata;
     const SDL_VideoData *viddata = (const SDL_VideoData *) _this->driverdata;
+#ifdef HAVE_LIBDECORATION_H
+    if (viddata->shell.libdecoration) {
+        SDL_SetError("FIXME libdecoration: Implement toggling decorations");
+    } else
+#endif
```

Comment:
This isn't a non-negotiable requirement to get this PR merged, but do we know if this FIXME is easy to implement, hard to implement, or impossible to implement, with the current state of libdecoration?

Reply:
There is a PR pending for this https://gitlab.gnome.org/jadahl/libdecoration/-/merge_requests/20, but it depends on `xdg-decoration` supporting this too https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/17.